src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const gallery = document.getElementById('thumbs');
+const largeImage = document.getElementById('largeImg');
+
+for (const picture of gallery.querySelectorAll('.list-item__link')) {
+  picture.addEventListener('click', (event) => {
```

Comment:
you can put a single event listener on the whole gallery instead of assigning a handler to all pictures (we should use [event delegation](https://learn.javascript.ru/event-delegation) in this task)